### PR TITLE
Create ncrypt.in.xml

### PR DESCRIPTION
**Pull request by heuser443**

### CODE REVIEW

**Comment by heuser443** [comment]
I need some advice how to fix the failing check here. The error message is unfortunately completely meaningless to me and I failed to find any documentation on it.

**Comment by heuser443** [comment]
Alright, the check now passed after an arbitrary change to the file. So it looks like the checking process is not deterministic here. #13706 likely has the same problem.

File: src/chrome/content/rules/ncrypt.in.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="ncrypt.in">
+	<target host="ncrypt.in" />
```
Comment (J0WI) [inline src/chrome/content/rules/ncrypt.in.xml]: Please also add `www.ncrypt.in`

**Comment by J0WI** [comment]
Please have a look at our policy for snapping redirects: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**